code/script.js

```diff
@@ -1,9 +1,61 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper')
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
+const handleNameInput = (event) => {
+event.preventDefault()
+
+  const name = nameInput.value
+  showMessage (name, 'user')
+  nameInput.value = ''
+
+  setTimeout(()=> showWebsiteOptions (name), 1000)
```

Comment:
When invoking a function, write the parentheses directly after (without space)
`showWebsiteOptions(name)`